test/powershell/engine/Cdxml/assets/CimTest/CdxmlTest.psd1

```diff
@@ -1,14 +1,14 @@
-@{
-    GUID = '41486F7D-842F-40F1-ACE4-8405F9C2ED9B'
-    Author="Microsoft Corporation"
-    CompanyName="Microsoft Corporation"
-    Copyright="� Microsoft Corporation. All rights reserved."
-    ModuleVersion = '2.0.0.0'
-    PowerShellVersion = '3.0'
-    FormatsToProcess = @()
-    TypesToProcess = @()
-    NestedModules = @( 'CimTest.cdxml')
-    AliasesToExport = @()
-    CmdletsToExport = @()
-    FunctionsToExport = @( 'Get-CimTest', 'Remove-CimTest', 'New-CimTest', 'Set-CimTest' )
-}
+@{
+    GUID = '41486F7D-842F-40F1-ACE4-8405F9C2ED9B'
+    Author="Microsoft Corporation"
+    CompanyName="Microsoft Corporation"
+    Copyright="(c) Microsoft Corporation. All rights reserved."
```

Comment:
Why we use defferent "c"? I see:
-  Copyright (c) 
-  Copyright (C) 
-  Copyright (©)

Reply:
I vaguely recall `(c)` is completely wrong, `(C)` has no legal weight, and that `©` should be used unless the intended process or system cannot parse it (in which case fall back to `(C)`. But I can't find a source.

Reply:
IANAL, but doing a search tells me the Berne Convention says none of those symbols are actually needed.  Let me follow up with our legal team to see what they recommend so we have consistency.  Based on prior experience, I don't expect them to get back to me very quickly, so I suggest we have this as a separate issue https://github.com/PowerShell/PowerShell/issues/4917

Reply:
We can look CoreFX experience.